src/app/containers/CpsFeaturesAnalysis/index.test.jsx

```diff
@@ -107,8 +115,44 @@ describe('CpsRelatedContent', () => {
     expect(asFragment()).toMatchSnapshot();
   });
 
-  it('should render Story Promo components without <ul> when given single item in collection', () => {
+  it('should render Story Promo components without <ul> when given single item in collection as the control variant', () => {
     isLive.mockImplementationOnce(() => true);
+    useOptimizelyVariation.mockReturnValue(null);
+
+    const topFeaturesOneItem = [features[0]];
+
+    expect(features[0]).toBeTruthy();
+
+    const { asFragment } = renderFeaturesAnalysis({
+      content: topFeaturesOneItem,
+    });
+
+    expect(document.querySelector(`li[class*='StoryPromoLi']`)).toBeNull();
+
+    expect(document.querySelector(`ul`)).toBeNull();
+
+    expect(asFragment()).toMatchSnapshot();
```

Comment:
I don't think we need this snapshot. I think snapshots should be separate tests too